content/kubernetes/security/manage-rec-certificates.md

```diff
@@ -25,23 +25,13 @@ To install and use your own certificates with Kubernetes on your Redis Enterpris
 
 ## Create a secret to hold the new certificate
 
-1. [Create the secret config file](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-config-file/) with the required fields shown below.
-
-    ```yaml
-    apiVersion: v1
-    kind: Secret
-    type: Opaque
-    metadata:
-      name: <secret-name>
-    data:
-      name: { proxy | api | cm | syncer | metrics_exporter } 
-      certificate: <certificate-string>
-      key:  <key-string>  
-    ```
+1. Create the [secret](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/) with the required fields shown below.
```

Comment:
Since there is only one step under this header now, maybe you can just remove the numbered step.
```suggestion
Create the [secret](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/) with the required fields shown below.
```